content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -401,17 +433,22 @@ mounted to the appropriate mount points.
 
 ## Scaling a StatefulSet
 
-Scaling a StatefulSet refers to increasing or decreasing the number of replicas.
+Scaling a StatefulSet refers to increasing or decreasing the number of replicas
+(horizontal scaling).
 This is accomplished by updating the `replicas` field. You can use either
 [`kubectl scale`](/docs/reference/generated/kubectl/kubectl-commands/#scale) or
 [`kubectl patch`](/docs/reference/generated/kubectl/kubectl-commands/#patch) to scale a StatefulSet.
 
-### Scaling Up
+### Scaling up
+
+Scaling up means adding more replicas. If you balance work across the
```

Comment:
I would disagree to this definition.
My understanding is that horizontal scaling is about scaling in or scaling out;
while vertical scaling is about scaling up or scaling down.